modules/module/LoRAModule.py

```diff
@@ -12,13 +12,18 @@ class LoRAModule(metaclass=ABCMeta):
     lora_down: nn.Module
     lora_up: nn.Module
     alpha: torch.Tensor
+    dropout: Dropout
 
-    def __init__(self, prefix: str, orig_module: nn.Module | None, rank: int, alpha: float):
+    def __init__(self, prefix: str, orig_module: nn.Module | None, rank: int, alpha: float, dropout_pct: float = 0):
         super(LoRAModule, self).__init__()
+        if dropout_pct < 0 or dropout_pct > 1:
```

Comment:
This check isn't needed. The Dropout module already does the same check and throws a ValueError.